src/components/manage/Blocks/Video/VideoSidebar.stories.jsx

```diff
@@ -0,0 +1,60 @@
+import { injectIntl } from 'react-intl';
+import React from 'react';
+import VideoSidebarComponent from './VideoSidebar';
+import { RealStoreWrapper as Wrapper } from '@plone/volto/storybook';
+
+const IntlVideoSidebarComponent = injectIntl(VideoSidebarComponent);
+
+function StoryComponent(args) {
+  return (
+    <Wrapper
+      customStore={{
+        intl: {
+          locale: 'en',
+          messages: {},
+        },
+      }}
+    >
+      <div id="toolbar" style={{ display: 'none' }} />
+      <IntlVideoSidebarComponent
+        {...args}
+        data={{}}
+        properties={{
```

Comment:
@Tishasoumya-02 I'm not sure this is the correct implementation. The preview_image comes from block schema in this case but not from content-type metadata. Also a storybook for Video block is sufficient in this case instead of Sidebar.